serbian/COPYING

```diff
@@ -0,0 +1,11 @@
+The file voc.txt was produced from the "srWaC – Serbian web corpus"
+version 1.0, downloaded from: http://nlp.ffzg.hr/resources/corpora/srwac/
+
+Java Application was used to parse tokens from corpus and create voc.temp.txt file.
+After that voc.temp.txt was manually reduced to 30000 tokens and saved to voc.txt.
```

Comment:
While this is OK, it would be more helpful if the Java code to reproduce this were available.  Just knowing it was written in Java is really no help at all to somebody in the future wanting to update the word list from a newer version of srWaC.